integration-cli/docker_cli_daemon_test.go

```diff
@@ -780,6 +780,18 @@ func (s *DockerDaemonSuite) TestDaemonDefaultGatewayIPv4Explicit(c *check.C) {
 	deleteInterface(c, defaultNetworkBridge)
 }
 
+func (s *DockerDaemonSuite) TestDaemonDefaultGatewayIPv4ExplicitOutsideContainerSubnet(c *check.C) {
```

Comment:
Confirmed the test fails if I checkout 5bdd4d0ec46ab071bbb633dde60b97b28911ade6^ (commit before #14966 's merge commit)